### DISCUSSION TITLE
IC-Light

### DISCUSSION THREAD

**Discussion by kyoushinosensei** [mashb1t/Fooocus#53] [Ideas]
Should we expect the addition of this functionality? As for me, it is very necessary and will make Fooocus even better!

**Comment by mashb1t** [comment]
IC-Light only works for SD1.5 (see [here](https://github.com/lllyasviel/IC-Light/blob/main/gradio_demo.py#L21)) and is based on diffusers. Fooocus is an SDXL only UI which doesn't use diffusers.
=> Not planned anytime soon.

**Reply by kyoushinosensei** [reply]
Very sad...((
